test/unit-test/core_sntp_config.h

```diff
@@ -32,13 +32,10 @@
 
 /************* Define Logging Macros using printf function ***********/
 
-#define PrintfError( ... )    printf( "Error: "__VA_ARGS__ );  printf( "\n" )
-#define PrintfWarn( ... )     printf( "Warn: "__VA_ARGS__ );  printf( "\n" )
-#define PrintfInfo( ... )     printf( "Info: " __VA_ARGS__ ); printf( "\n" )
-#define PrintfDebug( ... )    printf( "Debug: " __VA_ARGS__ ); printf( "\n" )
-
-/* Build with Debug level. */
-/*#define LOGGING_LEVEL_DEBUG    1 */
+#define PrintfError( ... )         printf( "Error: "__VA_ARGS__ );  printf( "\n" )
+#define PrintfWarn( ... )          printf( "Warn: "__VA_ARGS__ );  printf( "\n" )
+#define PrintfInfo( ... )          printf( "Info: " __VA_ARGS__ ); printf( "\n" )
+#define PrintfDebug( ... )         printf( "Debug: " __VA_ARGS__ ); printf( "\n" )
```

Comment:
Have you tested these definitions with parse_output.rb ?

Reply:
I have tested the logs in my local runs of unit tests. Here is an example snippet of the logs:
```
Debug: Server response has been validated: Server=my.ntp.server.1
Error: Unable to use server response: Server response failed sanity checks.
Debug: Server response has been validated: Server=my.ntp.server.1
Warn: Failed to calculate clock offset: System time SHOULD be within 34 years of server time.
Debug: Updating system time: ServerTime=2864434397 66ms ClockOffset=1000s
/home/ubuntu/Repos/coreSNTP/test/unit-test/core_sntp_client_utest.c:815:test_ReceiveTimeResponse_Deserialization_Failures:PASS
Debug: No data available on the network to read.
Debug: No data available on the network to read.
Debug: No data available on the network to read.
Debug: Server response has been validated: Server=my.ntp.server.1
Debug: Updating system time: ServerTime=2864434397 66ms ClockOffset=1000s
Debug: Server response has been validated: Server=my.ntp.server.1
Debug: Updating system time: ServerTime=2864434397 66ms ClockOffset=1000s
Debug: Updating system time: ServerTime=2864434397 66ms ClockOffset=1000s
/home/ubuntu/Repos/coreSNTP/test/unit-test/core_sntp_client_utest.c:915:test_ReceiveTimeResponse_Nominal:PASS
/home/ubuntu/Repos/coreSNTP/test/unit-test/core_sntp_client_utest.c:984:test_StatusToStr:PASS

-----------------------
9 Tests 0 Failures 0 Ignored 
OK
```